src/templates/blog-post.js

```diff
@@ -28,6 +28,15 @@ const MarkdownLayout = styled(Markdown)`
     margin-top: 0;
     padding-top: 0;
   }
+
+  & > p > code {
```

Comment:
src/component/Markdown/index.js has component 'Code' and it's applying styles on Prism from react-syntax-highlighter/dist/esm/styles/prism. Have you tried extending or copying then modifying those styles?

Reply:
Thanks! After taking a look at `react-syntax-highlighter/dist/esm/styles/prism/a11y-dark.js`, I see there are styles applied to the `code` block if its parent is not `pre`:
```
  ":not(pre) > code[class*=\"language-\"]": {
    "background": "#2b2b2b",
    "padding": "0.1em",
    "borderRadius": "0.3em",
    "whiteSpace": "normal"
  },
```

> Have you tried extending or copying then modifying those styles?

I copied the styles in the block above, then added `color: #f8f8f2`.  Is this what you are suggesting to do? Here's what it looks like now:
<img width="1025" alt="Screen Shot 2021-05-17 at 11 55 52 AM" src="https://user-images.githubusercontent.com/11492092/118541726-e3889d00-b706-11eb-8b18-4030b2e726aa.png">

Reply:
No. Basically you're hacking the css to work around some styles defined in an applied theme. Instead I'm suggesting it would be better, cleaner, and more supportable to actually define a markdown theme by maybe starting with an existing one and modifying it to suit your needs. An override hack like this can be a maintenance issue.

Reply:
Thanks for the explanation. Yes, I agree it is hacky and is not a good solution. I copied over the styles from `react-syntax-highlighter/dist/esm/styles/prism/a11y-dark.js` into `markdownTheme.js` then added a `code` CSS object to style the inline code.